autoload/git_commit_prefix.vim

```diff
@@ -16,8 +16,25 @@ let s:git_commit_prefix_candidates = [
   \ {'word': 'perf: ', 'menu': 'A code change that improves performance'},
   \ {'word': 'test: ', 'menu': 'Adding missing or correcting existing tests'},
   \ {'word': 'chore: ', 'menu': 'Changes to the build process or auxiliary tools and libraries such as documentation generation'}]
+
+let s:git_commit_prefix_candidates_ja = [
+  \ {'word': 'feat: ', 'menu': '新機能'},
```

Comment:
I would like to set the prefix candidate to DRY(Don't Repeat Yourself), is that possible?
If it is difficult, it's OK to keep this implementation.

Reply:
I guess I can, but sorry, now I have no time to refactor it.